infrastructure/lookerstudio/README.md

```diff
@@ -1,7 +1,16 @@
-# Marketing Analytics Looker Studio Dashboard
+# Marketing Data Engine Looker Studio Dashboard
 
+This Python script generates a URL to deploy a copy of the Marketing Data Engine dashboard utilizing the [Looker Studio Linking API](https://developers.google.com/looker-studio/integrate/linking-api).
 
+The dashboard deployment is intended to be completed once the Marketing Data Engine warehouse has been deployed through Dataform.
+
+The libraries required are in requirements.txt.
+
+You will need to configure config.ini with the BigQuery project and datasets for your warehouse.
 
 ## Run the walkthrough tutorial
 
+Click below to open the deployment tutorial in the Google Cloud console.
+
+[![Open in Cloud Shell](http://gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/editor?cloudshell_git_repo=https://github.com/TXZebra/marketing-data-engine&cloudshell_tutorial=infrastructure/lookerstudio/lookerstudio-deployment-walkthrough.md)
```

Comment:
The URL has a personal account id (TXZebra).

Reply:
My mistake. This will be fixed or removed depending upon what we end up doing.

Reply:
better remove, according to my last comment in the other thread.

Reply:
Done as part of refactor.